pkg/apis/eksctl.io/v1alpha5/validation_test.go

```diff
@@ -177,6 +185,7 @@ var _ = Describe("ClusterConfig validation", func() {
 			ng0 := cfg.NewNodeGroup()
 			ng0.Name = "node-group"
 			ng0.AMI = "ami-1234"
+			ng0.AMIFamily = api.DefaultNodeImageFamily
```

Comment:
should we make the vars here consistent as well?

Reply:
```suggestion
			ng0.AMIFamily = api.NodeImageFamilyAmazonLinux2
```